### ISSUE TITLE
vault read -field is somehow interpreting the value

### ISSUE THREAD

**Issue by saikatguha** [hashicorp/vault#2597]
## Steps to reproduce
1. `$ vault write secret/foo bar='%2b'`
2. `$ vault read secret/foo`
    ```
    Key                     Value
    ---                     -----
    bar                     %2b
    ```
3. `$ vault read -field=bar secret/foo`
    ```
    %!b(MISSING)
    ```

## Actual results
`%!b(MISSING)`

## Expected results
`%2b`

##

**Comment by vishalnayak** [comment]
@saikatguha I am not able to reproduce this on master. Which version of Vault are you using?

**Comment by jefferai** [comment]
Duplicate of #2109, fixed in 0.6.3.
